src/test/java/picard/analysis/directed/CollectHsMetricsTest.java

```diff
@@ -29,20 +29,25 @@ public String getCommandLineProgramName() {
     public Object[][] targetedIntervalDataProvider() {
         final String referenceFile = TEST_DIR + "/chrM.fasta";
         final String intervals = TEST_DIR + "/chrM.interval_list";
+        final String halfIntervals = TEST_DIR + "/chrM_100bp.interval_list";
         final String twoSmallIntervals = TEST_DIR + "/two-small.interval_list";
 
         return new Object[][] {
                 // two reads, each has 100 bases. bases in one read are medium quality (20), in the other read poor quality (10).
                 // test that we exclude half of the bases
```

Comment:
The comments for all the tests are now slightly incorrect because you've added in a new interval list `halfIntervals`. See comment on line `44 | 49` which explains what the auxiliary interval list `twoSmallIntervals` does.

Could you add on line `36 | 37` an additional statement on what `halfIntervals` is and why it's used?

Reply:
Added clarification for new tests